README.md

```diff
@@ -7,39 +7,75 @@
 
 ## Samples
 
-ForgeRock provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of ForgeRock and are not officially supported.
+Ping provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-To try out the ForgeRock JavaScript SDK please look at one of our samples:
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
 
-- [**Embedded login - `embedded-login`**](./embedded-login/README.md)
+To try out the Ping JavaScript SDK please look at one of our samples:
```

Comment:
I would suggest create a new category for each platform. Example:
## JavaScript
## iOS

Not a big deal. It's ok leave as it is.